primeqa/ir/dense/colbert_top/colbert/training/training.py

```diff
@@ -119,44 +100,14 @@ def train(config: ColBERTConfig, triples, queries=None, collection=None):
             teacher_reader = EagerBatcher(config, config.teacher_triples, (0 if config.rank == -1 else config.rank), config.nranks)
 
     if not config.reranker:
-        colbert = ColBERT(name=config.model_type, colbert_config=config)
-
-        # add support pre-trained representation
-        if config.init_from_lm is not None and config.checkpoint is None:
```

Comment:
Is loading from existing LM still supported?

Reply:
What do you mean by existing LM? The code supports models on huggingface and models stored locally.